config/routes.rb

```diff
@@ -41,6 +41,7 @@
       get 'submission_status/:job_id', to: 'disability_compensation_forms#submission_status', as: 'submission_status'
       get 'user_submissions'
       get 'suggested_conditions'
+      get 'find_rating_info_pid'
```

Comment:
i'd order these alphabetically if there's not any importance in their order

Reply:
Done!